src/sockets/commands/admin/areset.ts

```diff
@@ -1,30 +1,105 @@
 import { Command } from '../types';
 import { SocketUser } from '../../types';
 import User from '../../../models/user';
-import seasonNumber from '../../../util/seasonNumber';
+import RankData from '../../../models/rankData';
+import SeasonNumber from '../../../models/seasonNumber';
 
+// Return the sigmoid of the rating value
+function sigmoid(rating: number) {
+  return 1 / (1 + Math.exp(-rating));
+}
+
+// calculate the new rating value based on the old rating value
+function mapToRange(rating: number, minValue: number, maxValue: number) {
+  const midValue = (minValue + maxValue) / 2;
+  const sigmoided = sigmoid((rating - midValue) / 200);
+  return Math.floor(minValue + (sigmoided * (maxValue - minValue)));
+}
+
+// define a function to get the current season number
+async function getSeasonNumber() {
+  try {
+    const returnedSeasonNumber = await SeasonNumber.findOne({}).exec();
+    return returnedSeasonNumber.number;
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+// define a function to increment the season number
+function seasonNumberIncrement() {
+  SeasonNumber.findOne({}).exec().then(returnedSeasonNumber => {
+    returnedSeasonNumber.number++;
+    returnedSeasonNumber.save();
+  }).catch(err => {
+    console.error(err);
+  });
+}
```

Comment:
Yeah let's move this out of here and into some shared location along with getSeasonNumber().